docs/website/static/support.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
```

Comment:
Instead of copying all this stuff in here as a static html page can we re-use the hugo templates that build the main index.html page? Check out:

https://github.com/open-policy-agent/opa/tree/master/docs/website/layouts/partials/home
and
https://github.com/open-policy-agent/opa/blob/master/docs/website/layouts/index.html

Reply:
So, last week I had re-used the Hugo templates like you suggested, but @tsandall let me know that updates to "versioned" _content_ will not show up on the current website (will only show up on edge). He suggested I instead make a separate page (styled like the homepage) with the company list. Do you know if there is a way to use re-use the Hugo templates _without_ creating versioned content?